osu.Game/Skinning/Editor/SkinEditor.cs

```diff
@@ -266,14 +255,10 @@ private void placeComponent(Type type)
 
         private void populateSettings()
         {
-            settingsToolbox.Clear();
-
-            var first = SelectedComponents.OfType<Drawable>().FirstOrDefault();
+            settingsSidebar.Clear();
 
-            if (first != null)
-            {
-                settingsToolbox.Children = first.CreateSettingsControls().ToArray();
-            }
+            foreach (var component in SelectedComponents.OfType<Drawable>())
+                settingsSidebar.Add(new SkinSettingsToolbox(component));
```

Comment:
Not sure about creating these unconditionally like this even despite the skin component potentially having no settings to change. This was sorta kinda the case on `master` too but it was less egregious since it was just empty, but with this it's a bit weird to see a bunch of empty sections.

Reply:
I found it quite nice to be able to visualise what is selected, but can be swayed either way.

Reply:
It's not too egregious imo until we get more feedback. Will merge as is to keep things moving along.